docs/geedocs/5.3.2/raw_sources/answer/releaseNotes/relNotesGEE5_3_2.rst.txt

```diff
@@ -80,9 +85,27 @@ Release notes: Open GEE 5.3.2
            - Quitting 32-bit Windows portable server in Windows 7 via Internet Explorer does not work
            - IE appears to perform aggressive caching, which was preventing the GET request to cmd=quit
              from making it to the server. Changing the command to be a POST request prevents this.
-         * - 1394
-           - Upgrade PostgreSQL and PostGIS to latest patch releases
-           - The current patch releases have been incorporated.
+         * - 1376
+           - Portable Globe Cutter fails the last step when using HTTPS
+           - Used the advanced configuration to set a URL that will be used when cutter tries to make an openurl call.            
+         * - 1398
+           - System manager crashed because of casting asset version to asset

```

Comment:
This is too low level. For something like this we should state that it crashes and under what circumstances, if it can be determined. If it isn't, just say "Fixed System Manager crash".

Reply:
> This is too low level. For something like this we should state that it crashes and under what circumstances, if it can be determined. If it isn't, just say "Fixed System Manager crash".

Fixed